docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -937,11 +983,14 @@ public class ProtectedResource {
 }
 ----
 
-Note that `@TestSecurity` annotation must always be used and its `user` property is returned as `JsonWebToken.getName()` and `roles` property - as `JsonWebToken.getGroups()`.
-`@OidcSecurity` annotation is optional and can be used to set the additional token claims, as well as `UserInfo` and `OidcConfigurationMetadata` properties.
-Additionally, if `quarkus.oidc.token.issuer` property is configured then it will be used as an `OidcConfigurationMetadata` `issuer` property value.
+[NOTE]
```

Comment:
I'm not sure just because the text contains a `Note` it has to be turned into a dedicated note...

Reply:
@sberyozkin I changed it so that it is not a dedicated note, if you prefer. I modified the language slightly though to be more direct. Please see updated section.